src/components/proofs/ProofList.tsx

```diff
@@ -1,45 +1,54 @@
 import { AccentText, BodyText } from 'components/Text'
-import { Suspense } from 'preact/compat'
-import { space } from 'classnames/tailwind'
+import { CategoriesTitles, categories } from 'models/Categories'
+import { Suspense, useState } from 'preact/compat'
+import {
+  classnames,
+  display,
+  flex,
+  flexDirection,
+  gap,
+  space,
+} from 'classnames/tailwind'
 import { useSnapshot } from 'valtio'
-import ERC721ProofSection from 'components/proofs/ERC721ProofSection'
-import EmailProof from 'components/proofs/EmailProof'
-import Network from 'models/Network'
-import ProofSection from 'components/ProofSection'
+import DesktopProofCategories from 'components/proofs/DesktopProofCategories'
+import MobileProofCategories from 'components/proofs/MobileProofCategories'
 import ProofStore from 'stores/ProofStore'
-import ReadyEmailProof from 'components/proofs/ReadyEmailProof'
 import Scrollbar from 'components/Scrollbar'
 import WalletStore from 'stores/WalletStore'
 
+const proofList = classnames(
+  flex('flex-1'),
```

Comment:
For some reason, the proof list won't expand without this. `flex-1` is basically `flex` with configured `flex-shrink: 1` and `flex-grow: 1` + `flex-basis: 0%`

Without flex-1:
![image](https://user-images.githubusercontent.com/49658988/187159111-58203ebb-565f-4922-a1c2-ced3f4899ce2.png)
![image](https://user-images.githubusercontent.com/49658988/187159213-7a1d1c92-f459-48dd-8cf5-a2c1e2a73e87.png)